semperfiwebdesign/all-in-one-seo-pack
# Fix array_key_exists() expects array parameter notice in aioseop-class.php

Issue #2781

## Proposed changes

Fixes an issue where array_key_exists() does not receive an array as argument (in aioseop-class.php).

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Checklist


## Testing instructions

@wpsmort as discussed on Skype. Just install the PR and make sure this fixes the issue with BuddyPress forum profiles. Also check that this doesn't affect other post types and taxonomies.
